app/views/clients/_service_summary.haml

```diff
@@ -24,6 +24,12 @@
         .client__days-homeless
           = "#{@client.processed_service_history&.days_homeless_last_three_years || 0} homeless"
           %i.icon-info{data: {toggle: :tooltip, title: 'Days in ES, SO, SH, or TH with no overlapping PH after move-in date'}}
+        .client__days-homeless
+          = "#{@client.sheltered_days_homeless_last_three_years || 0} sheltered"
```

Comment:
won't `@client.sheltered_days_homeless_last_three_years` and `@client.unsheltered_days_homeless_last_three_years` return zero if there are none?